build-cluster-ec2.yml

```diff
@@ -125,3 +127,19 @@
   tasks:
     - include: "bundle/tasks/install-conductr-haproxy.yml"
 
+- name: Register ConductR with ELB
+  hosts: nodes
+  user: "{{ REMOTE_USER }}"
+  gather_facts: True
+  sudo: True
+
+  vars:
+# Get these from env to workaround https://github.com/ansible/ansible/issues/10638
```

Comment:
I've decided to leave reading AWS credentials as-is. If this is removed, it seems I have authentication problem intermittently:

```
TASK [Register with ELB] *******************************************************
fatal: [54.152.100.200 -> localhost]: FAILED! => {"changed": false, "failed": true, "msg": "No handler was ready to authenticate. 1 handlers were checked. ['HmacAuthV4Handler'] Check your credentials"}
fatal: [54.173.130.107 -> localhost]: FAILED! => {"changed": false, "failed": true, "msg": "No handler was ready to authenticate. 1 handlers were checked. ['HmacAuthV4Handler'] Check your credentials"}
fatal: [54.210.49.251 -> localhost]: FAILED! => {"changed": false, "failed": true, "msg": "No handler was ready to authenticate. 1 handlers were checked. ['HmacAuthV4Handler'] Check your credentials"}
	to retry, use: --limit @build-cluster-ec2.retry

PLAY RECAP *********************************************************************
127.0.0.1                  : ok=8    changed=6    unreachable=0    failed=0
54.152.100.200             : ok=55   changed=43   unreachable=0    failed=1
54.173.130.107             : ok=55   changed=43   unreachable=0    failed=1
54.210.49.251              : ok=61   changed=46   unreachable=0    failed=1
```

This is despite the fact that AWS credentials is available as environment vars (whose value I've removed):

```
ubuntu@ip-10-0-0-103:~/fsat/conductr-ansible$ env | grep -i aws
AWS_SECRET_ACCESS_KEY=<...>
AWS_ACCESS_KEY_ID=<...>
```